### ISSUE TITLE
Remove "reset password" from Login page

### ISSUE THREAD

**Issue by GunFood** [grafana/grafana#97620]
### What happened?

https://github.com/grafana/grafana/issues/17443

is still present. We have LDAP and soon oAuth (not SSO) as Backends enabled. We need an option to remove the password reset feature on the login page and from the user account. Maybe a "hide_password_reset" option in grafana.ini could do the required change, so no code change is needed.

### What did you expect to happen?

No "Passwort vergessen?" ("forgot password?") option on login page.

### Did this work before?

Seems in Version 6

### How do we reproduce it?

1. Install Grafana
2. Create LDAP Login connection
3. Check Login page

### Environment (with versions)?

Grafana: 11.3.1
OS: RedHat 8
Browser: any

### Grafana platform?

None

**Comment by GunFood** [comment]
disable_login = true is enabled now and "forgot password" is now gone. This is not helpful, as we have a mix with external customers and employees. We still need an option to remove the option even if internal authentication is used.

**Comment by kalleep** [comment]
Okey just so I get this right, you have and want both ldap and builtin login enabled and want to disable forgot password anyway?

I closed the issue because you stated that you only had LDAP and soon OAuth enbaled and only asked for a way to disable the forgot password feature. 

But if this is what you ask for it would require an additional flag, and updates in various places to disable it.

**Comment by GunFood** [comment]
No, I stated we have ldap activated, not it is the only authentication method.

I think it should be a configuration option "disable_sspr" and this value is checked at the login page and within the user menu. Just two places. But I'm not a programmer...

**Comment by Jguer** [comment]
Hey @GunFood, we are currently not planning to implement this feature. When internal login storage is enable we consider Grafana a full fledged IDP and SSPR an important feature of that functionality. We are committed to reducing the code paths of authentication as much as possible to avoid potential vulnerabilities.

I'm sorry this does not match your deployment needs, as a workaround the best I can think of is having a separate instance for internal users and utilizing stateless configuration for both internal and external instance

**Comment by wu-clan** [comment]
https://github.com/grafana/grafana/pull/114614
